magento/magento2
# Fix #6310 - Changing products 'this item has weight' using 'Update Attributes' is not possible

### Description (*)

This is fix for #6310

### Related Pull Requests
https://github.com/magento/partners-magento2ee/pull/150


### Fixed Issues (if relevant)

1. magento/magento2#6310: Changing products 'this item has weight' using 'Update Attributes' is not possible

### Manual testing scenarios (*)

1. Select a couple of virtual products in the product grid in the backend
2. Choose 'Update Attributes' in the 'Actions' dropdown
3. Search for the weight field, you'll find it
4. Search for the 'Does this has a weight?', you should be able to find it and use to change multiple products type at once

### Questions or comments


### Contribution checklist (*)
